.github/workflows/ci.yml

```diff
@@ -108,14 +110,32 @@ jobs:
       if: matrix.type.name == 'Debug' && runner.os != 'Windows' # Coverage is already generated on Windows when running tests.
       run: gcovr -r $GITHUB_WORKSPACE -x $GITHUB_WORKSPACE/build/coverage.out -s -f 'src/SFML/.*' -f 'include/SFML/.*' ${{ matrix.platform.gcovr_options }} $GITHUB_WORKSPACE
 
+    - name: Download Codecov Uploader
+      if: matrix.type.name == 'Debug' && github.repository == 'SFML/SFML' && env.HAVE_CODECOV_TOKEN == 'true' # Disable upload in forks and when Codecov token isn't available
```

Comment:
> and when Codecov token isn't available

Why would the token not be available inside SFML/SFML itself?

Reply:
There can be obscure scenarios where secrets are not available even when working within the main repository. I've seen cases of jobs in the past week or 2 in which the Codecov uploader complained that a token wasn't provided and reverted to auto-detect. It really shouldn't happen, but as many things in life things that shouldn't happen still do. It would also make it easier for us to detect since 2 whole steps would be skipped in the log and this is easier to scan for than having to open up the upload step and check whether it took the token or not.